tests/v2/test_waku_message_digest.nim

```diff
@@ -57,8 +64,41 @@ suite "Waku Message - Deterministic hashing":
 
     ## Then
     check:
+      byteutils.toHex(pubsubTopic.toBytes()) == "2f77616b752f322f64656661756c742d77616b752f70726f746f"
+      byteutils.toHex(message.contentTopic.toBytes()) == "2f77616b752f322f64656661756c742d636f6e74656e742f70726f746f"
+      byteutils.toHex(message.payload) == "010203045445535405060708"
+      byteutils.toHex(message.meta) == "73757065722d736563726574"
```

Comment:
I don't think this is necessary. It adds more work in case we want to update the test case 🤷🏼